app/app.go

```diff
@@ -658,6 +664,7 @@ func NewExocoreApp(
 	app.RewardKeeper = *rewardKeeper.NewKeeper(appCodec, keys[rewardTypes.StoreKey], app.AssetsKeeper)
 	app.ExoSlashKeeper = slashKeeper.NewKeeper(appCodec, keys[exoslashTypes.StoreKey], app.AssetsKeeper)
 	app.AVSManagerKeeper = *avsManagerKeeper.NewKeeper(appCodec, keys[avsManagerTypes.StoreKey], &app.OperatorKeeper, app.AssetsKeeper)
+	app.TaskKeeper = taskKeeper.NewKeeper(appCodec, keys[taskTypes.StoreKey], taskTypes.AvsKeeperMock{})
```

Comment:
Please update the Mock to real avsKeeper

Reply:
fixed